modules/workers/nodepools.tf

```diff
@@ -156,3 +156,158 @@ resource "oci_containerengine_node_pool" "workers" {
     }
   }
 }
+
+resource "oci_containerengine_node_pool" "with_size_ignored" {
```

Comment:
Please rename this to something that has nodepool in it.

Reply:
renamed it to `workers_with_size_ignored` to be consistent with the name of the existing nodepool - `workers` (where the size is not ignored).